tests/pykafka/test_protocol.py

```diff
@@ -256,5 +256,95 @@ def test_offset_fetch_response(self):
         self.assertEqual(response.topics[b'emmett.dummy'][0].offset, 1)
 
 
+class TestGroupMembershipAPI(unittest2.TestCase):
+    maxDiff = None
+
+    def test_consumer_group_protocol_metadata(self):
+        meta = protocol.ConsumerGroupProtocolMetadata()
+        msg = meta.get_bytes()
+        self.assertEqual(
+            msg,
+            bytearray(b'\x00\x00\x00\x00\x00\x01\x00\ndummytopic\x00\x00\x00\x0ctestuserdata')
```

Comment:
Since bytes can be painful in Python, we could forego the idea of the utility and just use indenting to improve readability:

```python
xs = bytearray(
    b'\x00\x00\x00\x00\x00\x01\x00\n'
    b'dummytopic'
    b'\x00\x00\x00\x0c'
    b'testuserdata')
```

I split lines based on the data contained. It will make the most sense to split lines based on what makes sense in the protocol (i.e. number of bytes or on a sentinel byte).